nixos/modules/services/networking/ntpd.nix

```
@@ -88,6 +91,21 @@ in
         default = [];
       };
 
+      extraConfig = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Extra configuration directives that should be added to
+          <literal>ntp.conf</literal>
+        '';
+      };
+
+      package  = mkOption  {
+        type = types.package;
+        description = "Which NTP Package to use.";
+        example = literalExample ''pkgs.ntpsec'';
```

Nitpick: You don't need the double-single-quotes for a single line.
```suggestion
        example = literalExample "pkgs.ntpsec";
```